src/bp-groups/bp-groups-functions.php

```diff
@@ -4500,3 +4504,72 @@ function groups_can_user_manage_video_albums( $user_id, $group_id ) {
 
 	return $is_allowed;
 }
+
+function bb_groups_loop_members( $group_id = 0, $role = array( 'member', 'mod', 'admin' ) ) {
```

Comment:
@milanhirapra where is function doc block?

Reply:
This is group directory related changes

Reply:
@milanhirapra We need to add the functions doc block.

for e.g

```
/**
 * Reset Cover image for Social Groups when changes backend setting(width and height).
 *
 * @since [BBVERSION]
 *
 * @param mixed  $old_value The old option value.
 * @param mixed  $value     The new option value.
 * @param string $option    Option name.
 */
```

Reply:
Done